.github/actions/lint/run.sh

```diff
@@ -3,4 +3,10 @@ set -e
 
 source /opt/ros/${ROS2_DISTRO}/setup.bash
 ./setup.sh
-ament_${LINTER} src/
+if [[ "$LINTER" == "clang-tidy" ]]
+then
+    ./build.sh OFF
```

Comment:
Why do you build only for clang?

Reply:
clang-tidy relies on a file generated by CMake to run properly.